### PR TITLE
Added CI

### PR DESCRIPTION
**Pull request by Nyholm**
Sorry for the big PR. 

I noticed that this library does not support Symfony 5. So I updated composer.json and added tests to make sure you can install it on Symfony 5 and 4. 

Since version 3 is broken, I would recommend to release version 4 after this is merged.

### CODE REVIEW

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#on: [push, pull_request]
+#name: Static analysis
+
+#jobs:
+#  php-cs-fixer:
+#    name: PHP-CS-Fixer
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
```
Comment (Nyholm) [inline .github/workflows/static.yml:9]: I recommend you do uncomment these and run PHP-cs-fixer.

I recommend you to do that in another PR.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 vendor/*
 composer.lock
-.idea/*
\ No newline at end of file
+.idea/*
```
Comment (Nyholm) [inline .gitignore:3]: `.idea` should be in a global gitignore. 

This should be removed in another PR.

File: composer.json [hunk 3]
```diff
@@ -3,14 +3,22 @@
   "description": "A few helpers to ease functional API testing in Symfony",
   "license": "MIT",
   "require": {
-    "php": "^7.0",
-    "symfony/framework-bundle": "^3.1 || ^4.0",
-    "symfony/browser-kit": "^3.1 || ^4.0",
```
Comment (Nyholm) [inline composer.json:8]: We never used browser-kit (only indirectly)

File: src/RequestBuilder.php [hunk 4]
```diff
@@ -3,12 +3,12 @@
 namespace Nebkam\FluentTest;
 
 use LogicException;
-use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
```
Comment (Nyholm) [inline src/RequestBuilder.php:6]: `Symfony\Bundle\FrameworkBundle\Client` has been deprecated

File: tests/ResponseWrapperTest.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+use Nebkam\FluentTest\ResponseWrapper;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class ResponseWrapperTest extends TestCase
+{
+    public function testIsCreated()
+    {
+        $wrapper = new ResponseWrapper(new Response('', 201));
+        $this->assertTrue($wrapper->isCreated());
```
Comment (Nyholm) [inline tests/ResponseWrapperTest.php:16]: This is just an example tests. Please add more tests in the next PR =)

**Comment by Nyholm** [comment]
Ping

**Comment by nebkam** [comment]
Thanks for a large PR,  @Nyholm , but I have to spend some time to grasp it (haven't used CI on Github yet and have to setup some tests of my own) and that takes time, which under these circumstances is scarce for me. I'll get back to you on this one, as soon as I get a bit more time

**Comment by Nyholm** [comment]
Sure. Take your time. 

Have a look at the ci output to confirm correctness. Also feel free to make comments on the lines in this PR if you want me to explain them better.

**Review by aradoje — APPROVED** [review]

**Review by aradoje — APPROVED** [review]
:+1:

**Comment by nebkam** [comment]
Thanks @Nyholm @aradoje !
To tell you the truth, I didn't know version 3 was broken
I'm very happy about the CI setup and look forward to using tests more thoroughly from now on.
Thanks again))

**Comment by Nyholm** [comment]
Thank you for merging

File: .github/workflows/static.yml (hunk 1 above)
Comment (nebkam) [reply .github/workflows/static.yml:9]: I did it and got some errors - https://gist.github.com/nebkam/adcfdadbcd03e3b90858c6f98e6892c4
Got any hints? I've used PHPStan for static analysis before, but a dockerized PHP CS Fixer is a first for me

Comment (Nyholm) [reply .github/workflows/static.yml:9]: See #8
